pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -554,6 +554,9 @@ func (k *Bootstrapper) restartControlPlane(cfg config.ClusterConfig) error {
 		return nil
 	}
 
+	if err := k.stopKubeSystem(cfg); err != nil {
+		glog.Warningf("failed to stop kube system container before reconfiguring kubeadm, this might cause port conflicts sometimes : %v", err)
```

Comment:
Suggestion: `Failed to stop kube-system containers: port conflicts may arise: %v`